src/components/ActivityCard.tsx

```diff
@@ -50,52 +34,37 @@ export default function ActivityCard() {
             <MoreVertIcon />
           </IconButton>
         }
-        title="Activity title"
-        subheader="Date"
+        title={activityItem.username}
       />
       <CardMedia
         component="img"
-        height="290"
-        image={ defaultimg }
-        alt="activityalt"
+        height="390"
+        src={"assets/" + activityItem.image }
```

Comment:
See my prev. comment: https://github.com/Asejs/Activity-Planner-React/pull/2/files#r954550959
Would suggest to use string template instead of classic string concatenation.